### PR TITLE
search results with file metadata

### PR DESCRIPTION
**Pull request by kostasb**
Add reference for file metadata in Search

Closes https://github.com/xataio/mdx-docs/issues/258

### CODE REVIEW

File: 040-SDK/150-file-attachments.mdx [hunk 1]
```diff
@@ -494,6 +494,93 @@ records, _ := searchClient.Query(context.TODO(), xata.QueryTableRequest{
 
 </TabbedCode>
 
+### Search results with files metadata
```
Comment (snide) [inline 040-SDK/150-file-attachments.mdx:497]:
```suggestion
### File attachments within search results
```

File: 040-SDK/150-file-attachments.mdx [hunk 2]
```diff
@@ -494,6 +494,93 @@ records, _ := searchClient.Query(context.TODO(), xata.QueryTableRequest{
...
+
+Files metadata are also available by default in [search](/docs/sdk/search) results. All metadata fields including the `url`,`signedUrl` and `uploadUrl` are returned as part of matching documents, but not the file content. Note that the `url` value is accessible only when [enablePublicUrl](/docs/concepts/file-attachments#public-urls) is set to `true` in the file's configuration.
```
Comment (snide) [inline 040-SDK/150-file-attachments.mdx:499]:
```suggestion
File attachment metadata is also available by default in [search](/docs/sdk/search) results. All metadata fields including the `url`,`signedUrl` and `uploadUrl` are returned as part of matching documents, but not the file content. Note that the `url` value is accessible only when [enablePublicUrl](/docs/concepts/file-attachments#public-urls) is set to `true` in the file's configuration.
```

**Review by snide — APPROVED** [review]
It might be worthwhile to add this content in the search section as well? Possibly just adding it to the shape returned here, with a note about the `public` nature of the URL?

https://xata.io/docs/sdk/search#search-response-with-highlight-markup

**Comment by kostasb** [comment]
> It might be worthwhile to add this content in the search section as well? Possibly just adding it to the shape returned here, with a note about the `public` nature of the URL?
> 
> https://xata.io/docs/sdk/search#search-response-with-highlight-markup

Good point, adding a section in the search docs

**Comment by kostasb** [comment]
@snide how does this look? https://xata-docs-pr-259.vercel.app/docs/sdk/search#search-response-with-file-attachments

I didn't want to inflate the [highlight](https://xata-docs-pr-259.vercel.app/docs/sdk/search#search-response-with-highlight-markup) part too much by adding file metadata there, instead added a standalone section.

**Review by snide — APPROVED** [review]
Looks great!

**Pull request merged**